spec/template.dd

```diff
@@ -1082,6 +1082,23 @@ $(H2 $(LNAME2 function-templates, Function Templates))
         }
         ---
 
+    $(P Variadic Function Templates can have parameters with default values.
+        These parameters are always set to their default value in case of IFTI.
```

Comment:
Forgive my ignorance, but what is IFTI?

Reply:
Implicit Function Template Instantiation?

Reply:
Found it here: https://wiki.dlang.org/Commonly-Used_Acronyms  Maybe spell it out in the description, with the abbreviation in parens, and then use the abbreviation in the comments of the example.

Reply:
it's alreay defined above in https://github.com/timotheecour/dlang.org/blob/277c1ad53a725594c4bf1b86e53afcdbaef1b099/spec/template.dd#L999